### PR TITLE
[DCOS_OSS-5951] edits to quickstart

### PR DESCRIPTION
**Pull request by msdolbey**
## [DCOS_OSS-5951](https://jira.d2iq.com/browse/DCOS_OSS-5951)
Feedback/updates on the Quick Start page

## Description of changes being made
- Some updates on the quick start page, new image to reflect that Kommander doesn't say beta right there.

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

@jaxesn @catsouthard @russurquhart1 @cmastr could you take a look?

### CODE REVIEW

**Comment by msdolbey** [comment]
@catsouthard - ahh - right here?
https://github.com/mesosphere/dcos-docs-site/pull/2878/files#diff-af3e37c7b3ad4f50b04e9b6ef39a5b77R44

**Comment by juliangieseke** [comment]
hey matt, this one is a bit tricky to automatically resolve after the 1.1 beta reworks, mind doing it manually? 🙇

**Comment by msdolbey** [comment]
@juliangieseke - I can totally resolve them manually - is it go time? Are all the other changes all merged in? Should I push some updates?

File: pages/ksphere/kommander/latest/quick-start/index.md [hunk 1]
```diff
@@ -19,27 +19,31 @@ To download Konvoy with Kommander, see the [Download Konvoy](https://docs.d2iq.c
 Download the tarball to your local Downloads directory.
 
 For example, if you are installing on MacOS, download the compressed archive to the default `~/Downloads` directory.
-Afterward, extract the tarball to your local system by running the following command:
+Afterward, extract the tarball to your local system by running the following command (make sure to double check the name of the file that you downloaded and change konvoy-kommander_darwin.tar.bz2 accordingly):
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/quick-start/index.md:22]:
```suggestion
Afterward, extract the tarball to your local system by running the following command (ensure the name of the file that you downloaded, and change konvoy-kommander_darwin.tar.bz2 accordingly):
```

File: pages/ksphere/kommander/latest/quick-start/index.md [hunk 2]
```diff
@@ -19,27 +19,31 @@ To download Konvoy with Kommander, see the [Download Konvoy](https://docs.d2iq.c
...
 Copy the Konvoy package files to a directory in your user's `PATH` to ensure you can invoke the konvoy command from any directory.
 
-For example, copy the package to `/usr/local/bin/` by running the following command:
+For example, copy the package to `/usr/local/bin/` by returning to the directory that Kommander was extracted to, and running the following command: 
 
 ```
 sudo cp ~/Downloads/darwin/konvoy-kommander.tar.bz2/* /usr/local/bin/
 ```
 
+NOTE: The extracted folder may have a different name (such as `konvoy_dev`, or perhaps it was given a custom directory, or have the version affixed to the end of the directory name)
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/quick-start/index.md:36]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>The extracted folder may have a different name (such as `konvoy_dev`, or perhaps it was given a custom directory, or have the version affixed to the end of the directory name)</p>
```

File: pages/ksphere/kommander/latest/quick-start/index.md [hunk 3]
```diff
@@ -49,9 +53,9 @@ konvoy up
 
 ### Logging in with Username and Password
 
-After you provision your first Konvoy cluster, your username, password, and a URL to Konvoy will be printed to the command-line. Once in Konvoy, you should see a button labelled "Try Kommander Beta!". If not, check to ensure you've installed the Konvoy release that includes Kommander.
+After you provision your first Konvoy cluster, your randomly-generated username, password, and a URL to Konvoy will be printed to the command-line. Once in Konvoy, you should see a button labelled "Try Kommander". If not, check to ensure you've installed the Konvoy release that includes Kommander.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/quick-start/index.md:56]:
```suggestion
After you provision your first Konvoy cluster, your randomly-generated username, password, and a URL to Konvoy are printed to the command-line. Once in Konvoy, you should see a button labelled "Try Kommander". If not, ensure you've installed the Konvoy release that includes Kommander.
```

**Comment by msdolbey** [comment]
Closing pr because this one is outdated.

**Pull request closed without merging**